src/vs/workbench/contrib/chat/browser/tools/fileContentsTool.ts

```diff
@@ -28,14 +26,18 @@ export const FileContentsToolData: IToolData = {
 	displayName: localize('chat.tools.getFileContents', "Get File Contents"),
 	source: ToolDataSource.Internal,
 	modelDescription: getFileContentsModelDescription,
-	tags: ['positron-assistant', 'requires-workspace'],
+	tags: [
+		'positron-assistant',
+		'requires-workspace',
+		'high-token-usage',
+	],
 	canBeReferencedInPrompt: false,
 	inputSchema: {
 		type: 'object',
 		properties: {
 			filePath: {
 				type: 'string',
-				description: 'The file path to get the contents of.',
+				description: 'The absolute file path to get the contents of. The file path must be a path to a file in the workspace or a file that is currently open in the editor.',
```

Comment:
Requiring absolute file paths seems to resolve the "file doesn't exist" type of errors.

Also moved input param-specific guidance to the param description.